test/torchaudio_unittest/datasets/vctk_test.py

```diff
@@ -0,0 +1,104 @@
+import os
+
+from torchaudio.datasets import vctk
+
+from torchaudio_unittest.common_utils import (
+    TempDirMixin,
+    TorchaudioTestCase,
+    get_whitenoise,
+    save_wav,
+    normalize_wav,
+)
+
+
+# Used to generate a unique utterance for each dummy audio file
+UTTERANCE = [
+    'Please call Stella',
+    'Ask her to bring these things',
+    'with her from the store',
+    'Six spoons of fresh snow peas, five thick slabs of blue cheese, and maybe a snack for her brother Bob',
+    'We also need a small plastic snake and a big toy frog for the kids',
+    'She can scoop these things into three red bags, and we will go meet her Wednesday at the train station',
+    'When the sunlight strikes raindrops in the air, they act as a prism and form a rainbow',
+    'The rainbow is a division of white light into many beautiful colors',
+    'These take the shape of a long round arch, with its path high above, and its two ends \
+        apparently beyond the horizon',
+    'There is, according to legend, a boiling pot of gold at one end'
+]
+
+
+class TestVCTK(TempDirMixin, TorchaudioTestCase):
+    backend = 'default'
+
+    root_dir = None
+    samples = []
+
+    _ext_txt = ".txt"
+    _ext_audio = ".wav"
+    _folder_txt = "txt"
+    _folder_audio = "wav48_silence_trimmed"
```

Comment:
None of these need to be variable or class attributed. Can you just embed in `setUpClass` function?

Reply:
modified accordingly